jvb/src/main/java/org/jitsi/videobridge/cc/allocation/SingleSourceAllocation.java

```diff
@@ -129,7 +129,8 @@
                     || (lessThanOrEqualIdealResolution && atLeastPreferredFps))
                     || ratesList.isEmpty())
             {
-                double layerBitrate = layer.getBitrate(nowMs);
+
+                double layerBitrate = layer.getBitrateBps(nowMs);
```

Comment:
Are these code changes suppose to go into this PR?

Reply:
Yes, they are required because with the updated jitsi-media-utils`RtpLayerDest.getBitrate()` doesn't get exported and can't be used in java (I don't know why, but this was the workaround I found).